### DISCUSSION TITLE
Upload Questions in batches

### DISCUSSION THREAD

**Discussion by zhenduowen** [PrairieLearn/PrairieLearn#8653] [Q&A] [answered]
Hi, 

Is there a way to upload questions in batches? We are trying to transfer from another learning platform EdX to Prairielearn. I have implemented a converter to align the question format with the one we use on Prairielearn. The only issue for me is that I can only upload homework questions one by one and we have many questions. For now, I only know how to create questions in the browser. Is there a way to upload questions to my course on Prairielearn in batches?

Many thanks.

**Comment by mwest1066** [comment, marked as answer]
Yep! PrairieLearn courses are backed by `git` repositories. You can check out the repo, add all your questions, and then push them in one go. Can you paste the URL for your course here? We can then tell you how to access the backing `git` repo.

**Reply by zhenduowen** [reply]
Thanks! The student link for this course is https://us.prairielearn.com/pl/course_instance/142217 And the GitHub repo for our course is git@github.com:PrairieLearn/pl-ucsd-ece225.git

**Reply by mwest1066** [reply]
Great! Would you be able to send me an email from your `@ucsd.edu` email address confirming this request? This will serve as confirmation of your identity and role in the course.

**Reply by mwest1066** [reply]
You can send me the email at matt@prairielearn.com

**Reply by zhenduowen** [reply]
Sure. The email was sent with the title Prairielearn Upload Questions in batches #8653
